views/play.erb

```diff
@@ -0,0 +1,14 @@
+<body style="background-color:black;">
+<h1 style="color:gold;"> Hi <%= @player %>, lets play Rock, Paper, Scissors</h1>
+
+<form action='/choserock' method="post"> 
```

Comment:
Following on from one page for results, use one form for your input

Reply:
And have also changed this. Thank you